### ISSUE TITLE
Adafruit NeoPixel Library 

### ISSUE THREAD

**Issue by SPERROWins** [mathertel/DmxSerial2#44]
If pixel.show() command is executed, I can no longer see my device on rdm. 
I assume the two library's are conflicting somehow (I'm really not good at code).

So my question is:
Is this true and if so is there an easy fix?

**Comment by fritzlb** [comment]
Yes, this is true and unfortunately there is not that much you can do. `pixel.show()` takes control of the neopixel pin and sends out the data, depending on the number of pixels and their color scheme (RGB or RGBW) this takes typically a few milliseconds. Because timing is critical, interrupts are disabled during this, so absolutely nothing else can be done simultaneously. Unfortunately, this includes DMX/RDM processing.
One thing you might try: limit the frequency in which pixel.show gets called. For example, check if 100ms have passed using millis(). Depending on your requirements, this might not suffice though. In that case, there is no real other option than to change from your uno to a different platform. For example, ESP is capable enough (it has a second core you could use for neopixel), but the esp_dmx library currently has its own quirks. 

Disclaimer: I do not know this library very well. But I do have some experience with driving neopixels through AVR ASM and bit-banging DMX on different platforms
